pom.xml

```diff
@@ -24,6 +24,7 @@
         <junit.jupiter.version>5.9.2</junit.jupiter.version>
         <solr.solrj.version>8.10.1</solr.solrj.version>
         <embedded-redis.version>0.7.3</embedded-redis.version>
+        <jackson.version>2.17.2</jackson.version>
```

Comment:
just curious what was happening that required this? I thought jackson.version would be available from boxc. Did this need a newer version?